sentry_sdk/tracing.py

```diff
@@ -2,14 +2,21 @@
 import uuid
 import contextlib
 import math
+import random
 import time
 
 from datetime import datetime, timedelta
+from inspect import isfunction
 from numbers import Real
 
 import sentry_sdk
 
-from sentry_sdk.utils import capture_internal_exceptions, logger, to_string
+from sentry_sdk.utils import (
+    capture_internal_exceptions,
+    has_tracing_enabled,
```

Comment:
I know this was part of a diff PR but can we move `has_tracing_enabled` into this module?